STMicroelectronics/STM32L4_L4+/app/cipher/cipher_v2i1/interfaces/cipher_v2i1_interface.c

```diff
@@ -121,37 +62,43 @@ static az_result cipher_1_encrypt_span_wrapper(az_span model_in_span, az_span* m
     cipher_1_encrypt_model_out encrypt_model_out = { .dest = &dest_span };
 
     // Call.
-    result = cipher_1_encrypt_concrete(
-        (az_ulib_model_in)&encrypt_model_in, (az_ulib_model_out)&encrypt_model_out);
+    AZ_ULIB_THROW_IF_AZ_ERROR(cipher_1_encrypt_concrete(
+        (az_ulib_model_in)&encrypt_model_in, (az_ulib_model_out)&encrypt_model_out));
 
-    // Marshalling model_out_span to JSON.
+    // Marshalling model_out to JSON.
```

Comment:
nit: you're still using model_out_span here

Reply:
There is a confusion in the comment, I'll fix that. The correct statement is 
```
// Marshalling encrypt_model_out to JSON in model_out_span
```